### PR TITLE
How to manage your inventory

### PR DESCRIPTION
**Pull request by safronenko23**
**⚠️Please do not make any changes to spec files in this repository!⚠️**

**All spec changes should be done in the [api-specs](https://github.com/Bandwidth/api-specs) repository.**

**Please follow the [Guide](https://bandwidth-jira.atlassian.net/wiki/spaces/DX/pages/4098359409/How+to+Update+API+Specifications+and+Contribute+to+Developer+Documentation) for contributing to our developer documentation.**

### CODE REVIEW

**Comment by safronenko23** [comment]
@ajrice6713 @rjamesq Any comments on this?

**Comment by safronenko23** [comment]
@gocab-bw I cannot add you as a reviewer

File: site/docs/numbers/manageInventory.mdx [hunk 1]
```diff
@@ -0,0 +1,1203 @@
+---
+id: manageInventory
+title: How to manage your inventory
+slug: /numbers/guides/searchNumbers
```
Comment (gocab-bw) [inline site/docs/numbers/manageInventory.mdx:4]: I would change the slug here to /numbers/guides/manageInventory unless it needs to be /numbers/guides/searchNumbers

Comment (ajrice6713) [reply site/docs/numbers/manageInventory.mdx:4]: We will need to setup a redirect for the URL so just ping me when this is going live

File: site/docs/numbers/manageInventory.mdx [hunk 2]
```diff
@@ -1197,6 +1206,947 @@ curl 'https://dashboard.bandwidth.com/api/tns/{tn}/availableNnRoutes'
...
+        </LidbTnGroups>
+    </LidbOrder>
+</LidbOrderResponse>
+```
+
+</TabItem>
+
+<TabItem value="dlda">
+
+The DLDA service allows the association of a name and address listing with a Telephone Number or Telephone Numbers in the Bandwidth Phone Number Phone Number system.  The DLDA API follows the asynchronous order processing request model described elsewhere in this document, where a request is submitted, an order ID is returned as an ID of that request or order, and then the request is fulfilled in the background.  The status of the request is reflected in the status of the order, and can be queried at any time.
```
Comment (gocab-bw) [inline site/docs/numbers/manageInventory.mdx:1759]: Change "The DLDA service allows the association of a name and address listing with a Telephone Number or Telephone Numbers in the Bandwidth Phone Number Phone Number system" to "The DL/DA API allows a telephone number or group of telephone numbers to be associated with a name and address used for Directory Listing and/or Directory Assistance information."

Change "DLDA API" to "DL/DA API"

File: site/docs/numbers/manageInventory.mdx [hunk 3]
```diff
@@ -1197,6 +1206,947 @@ curl 'https://dashboard.bandwidth.com/api/tns/{tn}/availableNnRoutes'
...
+
+To create a DLDA Order, make a <Highlight color="#079CEE">POST</Highlight> request to our [Create DLDA Order API](../../../../apis/numbers#tag/accounts/operation/CreateDldaOrder) endpoint.
```
Comment (gocab-bw) [inline site/docs/numbers/manageInventory.mdx:1761]: Change all instances of "DLDA" to "DL/DA"

File: site/docs/numbers/manageInventory.mdx [hunk 4]
```diff
@@ -1197,6 +1206,947 @@ curl 'https://dashboard.bandwidth.com/api/tns/{tn}/availableNnRoutes'
...
+This can be done through tools like [Postman](https://github.com/Bandwidth/postman) or in the command line.
```
Comment (gocab-bw) [inline site/docs/numbers/manageInventory.mdx:1762]: Change "in the command line" to "cURL"

File: site/docs/numbers/manageInventory.mdx [hunk 5]
```diff
@@ -1197,6 +1206,947 @@ curl 'https://dashboard.bandwidth.com/api/tns/{tn}/availableNnRoutes'
...
+
+Please follow the [How to setup Notification Webhook](/docs/numbers/webhooks/orderWebhook) guide for more information.
+
+<Tabs
+groupId="fetch_order_status"
+defaultValue="option_order"
+values={[
+    { label: 'TN Options Order', value: 'option_order', },
+    { label: 'Calling Name Display Update (LIDB Order)', value: 'lidb', },
+    { label: 'Directory Listing and Directory Assistance (DLDA Order)', value: 'dlda', }
```
Comment (gocab-bw) [inline site/docs/numbers/manageInventory.mdx:1955]: Change "DLDA Order" to "DL/DA Order"

File: site/docs/numbers/manageInventory.mdx [hunk 6]
```diff
@@ -1197,6 +1206,947 @@ curl 'https://dashboard.bandwidth.com/api/tns/{tn}/availableNnRoutes'
...
+
+:::tip POLL FOR ORDER STATUS
+These orders require manual intervention and take over 24 hours to process.
+:::
+
+</TabItem>
+
+<TabItem value="dlda">
+
+Since the DLDA order is asynchronous, it is recommended to create a dldas subscription to receive order status webhooks.
```
Comment (gocab-bw) [inline site/docs/numbers/manageInventory.mdx:1979]: Change "DLDA" to "DL/DA"

File: site/docs/numbers/manageInventory.mdx [hunk 7]
```diff
@@ -1197,6 +1206,947 @@ curl 'https://dashboard.bandwidth.com/api/tns/{tn}/availableNnRoutes'
...
+defaultValue="location"
+values={[
+{ label: 'Setup Failover on a Location (Sip-Peer)', value: 'location', },
+{ label: 'Setup Failover on a Single Number/Group of Numbers', value: 'single', }
+]
+}>
+<TabItem value="location">
+
+To setup a failover for a location (sip-peer), make a <Highlight color="#9a59c5">PUT</Highlight> request to our [Update SipPeer API](../../../../apis/numbers#tag/accounts/operation/UpdateSipPeer) endpoint.
+This can be done through tools like [Postman](https://github.com/Bandwidth/postman) or in the command line.
```
Comment (gocab-bw) [inline site/docs/numbers/manageInventory.mdx:2005]: Change "in the command line" to "cURL"

**Review by gocab-bw — CHANGES_REQUESTED** [review]

File: site/docs/numbers/manageInventory.mdx (hunk 4 above)
Comment (safronenko23) [reply site/docs/numbers/manageInventory.mdx:1762]: @gocab-bw I have been using this sentnce in previeus guids also. For example in `seach and order numbers`. Should I change it then?

File: site/docs/numbers/manageInventory.mdx (hunk 2 above)
Comment (gocab-bw) [reply site/docs/numbers/manageInventory.mdx]: I just wanted the intro to say "The DL/DA API allows a telephone number or group of telephone
numbers to be associated with a name and address used for Directory Listing and/or Directory Assistance information."  Looks like the original is still in there.

File: site/docs/numbers/manageInventory.mdx (hunk 4 above)
Comment (gocab-bw) [reply site/docs/numbers/manageInventory.mdx]: Yes, please change in other places.

Comment (safronenko23) [reply site/docs/numbers/manageInventory.mdx]: I've already changed them also

**Review by gocab-bw — DISMISSED** [review]

File: site/docs/numbers/manageInventory.mdx [hunk 8]
```diff
@@ -0,0 +1,2152 @@
...
+        backgroundColor: color,
+        borderRadius: '2px',
+        color: '#fff',
+        padding: '0.2rem',
+      }}>
+      {children}
+    </span>
+  );
+
+In this guide, we will show you how to manage inventory across your Bandwidth account using the number search, detail, and reservation APIs. Associating the number with “Option” or “Feature” characteristics.
```
[the remark this answers is not in the document]
Comment (rjamesq) [reply site/docs/numbers/manageInventory.mdx]: Awesome job on this guide. Would it be possible to break out the sub sections into sub pages under "How to manage your inventory" similar to how Paul structured the porting guide?

**Review by gocab-bw — APPROVED** [review]

**Review by rjamesq — APPROVED** [review]

**Pull request merged**